corehq/apps/users/views/mobile/users.py

```diff
@@ -1119,34 +1119,11 @@ def user_upload_job_poll(request, domain, download_id, template="users/mobile/pa
     context.update({
         'on_complete_short': _('Bulk upload complete.'),
         'on_complete_long': _('Mobile Worker upload has finished'),
+        'user_type': _('mobile workers'),
 
     })
 
-    class _BulkUploadResponseWrapper(object):
```

Comment:
Is this just moved as is?

Reply:
Yes-- a slight addition to distinguish between mobile and web but otherwise unchanged